src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -295,7 +294,7 @@ For many apps, user sign-up and sign-in with a username/password is all that is
 <Accordion title='Review the sign-up, sign-in, and sign-out workflow' headingLevel='3' eyebrow='Learn more'>
 
 Below is a high-level overview of the workflows for sign-up, sign-in, and sign-out with Amplify authentication:
-
+ 
```

Comment:
This would be a good place to add a diagram if we want to break up the text more. Happy to create it/them but may also be something we add in a later version?

Reply:
I think this is fine for now. Let us start moving the content around and we can decide then.

Reply:
Do we have a mechanism for the management of diagrams now?

Reply:
> Do we have a mechanism for the management of diagrams now?

No not atm. This would be part of that work which is another reason to leave this until later.